manifests/WiXToolset/WiXToolset/3.11.2.yaml

```diff
@@ -0,0 +1,17 @@
+Id: WiXToolset.WiXToolset
```

Comment:
This would change with the publisher, but not sure if a '.' is allowed at the start of the Id.  When I was testing mine I did "dotnet.foundation.wix.toolset"

Reply:
It seems PascalCasing is used by the vast majority of packages, so I chose DotNetFramework.